Code/src/main/java/nl/utwente/group10/ui/menu/CircleMenu.java

```diff
@@ -1,12 +1,12 @@
 package nl.utwente.group10.ui.menu;
 
-import nl.utwente.group10.ui.components.blocks.Block;
+import javafx.util.Duration;
 
+import nl.utwente.group10.ui.components.blocks.Block;
```

Comment:
> Do you need this import?

You mean the JavaFX duration? Yes I use it in line 60 the setAnimationDuration only accepts Duration values.